torchaudio/functional/functional.py

```diff
@@ -1444,3 +1446,82 @@ def resample(
                                               resampling_method, beta, waveform.device, waveform.dtype)
     resampled = _apply_sinc_resample_kernel(waveform, orig_freq, new_freq, gcd, kernel, width)
     return resampled
+
+
+def character_edit_distance(r: str, h: str) -> int:
+    """
+    Calculate the character level edit (Levenshtein) distance between two strings.
+
+    The function computes an edit distance allowing deletion, insertion and substitution.
+    The result is an integer. Users may want to normalize by the length of the reference.
+
+    This can be used to compute the edit distance for instance between two strings,
+    or two list of words. Note that, if a string and a list of words is provided, the distance
+    will be computed between the "string" sequence, and the "list of words" sequence.
+
+    Args:
+        r (str): the reference string to compare.
+        h (str): the hypothesis string (e.g. the predicted string) to compare.
+    Returns:
+        int: The distance between the reference and the hypothesis.
+    """
+
+    # Initialisation
```

Comment:
Please remove these comments, that are too obvious and do not provide any additional information.
The code should be self-explanatory.

Reply:
I'll remove them. Thanks :)